src/databricks/sql/client.py

```diff
@@ -244,7 +247,18 @@ def _close(self, close_cursors=True) -> None:
         if close_cursors:
             for cursor in self._cursors:
                 cursor.close()
-        self.thrift_backend.close_session(self._session_handle)
+        try:
+            logger.info(f"Closing session {self.get_session_id_hex()}")
+            self.thrift_backend.close_session(self._session_handle)
+        except DatabaseError as e:
+            if "Invalid SessionHandle" in str(e):
+                logger.warning(
+                    f"Attempted to close session that was already closed: {e}"
+                )
+                pass
+            else:
+                raise e
```

Comment:
wondering if we should swallow all exceptions for close session calls.

Reply:
Good thought. I've updated the code to log all exceptions but not raise them.